### ISSUE TITLE
Улучшенный симулятор

### ISSUE THREAD

**Issue by Flyrine** [SBProX/SBProX-Dev#229]
![Sim X](https://user-images.githubusercontent.com/51110220/58618082-e0166400-82eb-11e9-8519-d266cba43aca.jpg)

И это будет одним из лучших симуляторов торговых и аналитических платформ

**Comment by Flawmaster** [comment]
Еще было бы неплохо, чтобы кнопки управления симулятором(ускорение, пауза, стоп) были в одном окне с графиком, либо висели в другом окне в режиме "always on top". А то каждый раз приходится разворачивать, чтобы остановить/возобновить движение цены.

**Comment by Anton144** [comment]
> ![Sim X](https://user-images.githubusercontent.com/51110220/58618082-e0166400-82eb-11e9-8519-d266cba43aca.jpg)
> 
> И это будет одним из лучших симуляторов торговых и аналитических платформ

А ещё лучше будет если создать и совместить торговый модуль с панелью текущего баланса,картинку взял с прошлого тикета!
![4](https://user-images.githubusercontent.com/18700341/58625176-52d30f80-82eb-11e9-9fff-146d43a55fd1.png)

**Comment by vitalii13** [comment]
возможность сохранение сделок в файл, что бы потом можно было открыть этот файл в ексель

**Comment by Flyrine** [comment]
> А ещё лучше будет если создать и совместить торговый модуль с панелью текущего баланса,картинку взял с прошлого тикета!

Да,именно так в них и реализовано и панель баланса будет вобще не нужна

**Comment by bushevigor** [comment]
Поддерживаю

**Comment by Sergo52** [comment]
Можно сделать симулятор более плавный? Чтобы он шел побарно, а не по 5-10 баров. Это актуально когда просматриваешь несколько таймфреймов, например М15 и М1 для входа, М1 быстроскачет.

И можно сделать симулятор в реальном времени, а не только на истории. Т.е. возможность виртуально торговать в реальном времени

**Comment by Flyrine** [comment]
> Можно сделать симулятор более плавный? Чтобы он шел побарно, а не по 5-10 баров

Так он не идет по 5-10 баров и даже не по одному.Симулятор идет по получаемым им тикам от котировок там полная реальная плавность.Может у вас дело в скорости?Выставите 1
